doc/whats_new.rst

```diff
@@ -149,6 +149,10 @@ Enhancements
 
 Bug fixes
 .........
+   - Fixed a bug where :class:`sklearn.cluster.DBSCAN` gives incorrect 
+     result when input is precomputed sparse matrix with initial rows
+     all zero.
```

Comment:
I would have rather said "with all-zeros initial rows" but I am not a native English speaker and I don't know which is more correct.